data-structures-linked-lists/get-length.js

```diff
@@ -0,0 +1,9 @@
+export default function getLength(list) {
+  let data = list;
```

Comment:
This is safe, but there's no harm in using `list` here, even though you are modifying it. You will not modify the variable that is passed as an argument, so you don't need to worry about that. If you had wanted to return the original list, then making a copy is important, but that's not what this function does.